draft/root-transitions.md

```diff
@@ -0,0 +1,54 @@
+---
+RFC: XXXX
+Author: Kyle Fuller
+Status: Draft
+Created: 2015-09-21
+Last Modified: 2015-09-21
+---
+
+# API Blueprint RFC XXXX: Root Transitions
+
+## Table of Contents
+
+- [Abstract](#abstract)
+- [Motivation](#motivation)
+- [Proposed Syntax](#proposed-syntax)
+
+## Abstract
+
+This RFC proposes that you can define the transitions you may make from the
+root of an API described by API Blueprint.
```

Comment:
From these two lines, I seem to understand that you want to somehow define `_links` in the HAL object. But the second sentence in the `Motivation` seems to contradict this statement which is plain confusing me.